include/exadg/postprocessor/write_output.h

```diff
@@ -78,13 +81,60 @@ write_boundary_IDs(dealii::Triangulation<dim> const & triangulation,
 template<int dim>
 void
 write_grid(dealii::Triangulation<dim> const & triangulation,
+           dealii::Mapping<dim> const &       mapping,
+           unsigned int const                 n_subdivisions,
            std::string const &                folder,
-           std::string const &                file)
+           std::string const &                file,
+           unsigned int const &               counter,
+           MPI_Comm const &                   mpi_comm)
 {
-  std::string filename = folder + file + "_grid";
+  std::string filename = file + "_grid";
```

Comment:
I don't see the reason to append the filename here in these functions `_grid` (and `_points` below). Should we maybe change this in the future? It is somewhat independent of the present PR.

Reply:
I see the point, but this makes sure the files are not overwritten calling the output with identical file names ... otherwise we shift that to the calling functions ... I would be ok with both!